--- server/Makefile.in.orig	2009-07-27 13:28:29.000000000 +0200
+++ server/Makefile.in	2013-12-27 16:12:10.581542700 +0100
@@ -485,6 +485,7 @@
 	$(top_builddir)/libplayercore/libplayerutils.la \
 	$(top_builddir)/libplayertcp/libplayertcp.la \
 	$(top_builddir)/libplayertcp/libplayerudp.la \
+	$(top_builddir)/libplayerjpeg/libplayerjpeg.la \
 	$(top_builddir)/libplayerxdr/libplayerxdr.la $(am__append_1)
 
 # Note to everybody: please do NOT break the following assignment onto
@@ -498,6 +499,7 @@
 	$(top_builddir)/libplayercore/libplayerutils.la \
 	$(top_builddir)/libplayertcp/libplayertcp.la \
 	$(top_builddir)/libplayertcp/libplayerudp.la \
+	$(top_builddir)/libplayerjpeg/libplayerjpeg.la \
 	$(top_builddir)/libplayerxdr/libplayerxdr.la \
 	@PLAYER_DRIVER_EXTRA_LIBS@ $(RTKGUI_LIBS) \
 	$(top_builddir)/replace/libreplace.la @ARTOOLKITPLUS_LIBS@ \
