--- Imagemagick/configure.orig	2013-11-26 18:07:53.214390000 +0100
+++ Imagemagic/configure	2013-11-26 18:08:07.894390000 +0100
@@ -9432,7 +9432,7 @@
         if test "$cross_compiling" = yes; then :
   { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
 $as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
+as_fn_warning $? "cannot run test program while cross compiling
 See \`config.log' for more details" "$LINENO" 5; }
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
